<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>185</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Действия</string>
       </property>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>183</width>
          <height>26</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>182</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Ключ:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>182</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Значение:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>183</width>
          <height>26</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>183</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>insert</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>183</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>remove</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>183</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>findMinStack</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>183</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>230</y>
          <width>183</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
